sphinx/texinputs/sphinx.sty

```diff
@@ -710,64 +714,105 @@ will be set to white}%
-        \the\toks0\expandafter{%
-        \csname spx@opt@heavynotetrue\expandafter\expandafter\expandafter\endcsname
-        \csname KV@sphinx@div.note_#1\endcsname{##1}}%
-    \expandafter\def\csname KV@sphinx@div.hint_#1\expandafter\endcsname
-        \the\toks0\expandafter{%
-        \csname spx@opt@heavyhinttrue\expandafter\expandafter\expandafter\endcsname
-        \csname KV@sphinx@div.hint_#1\endcsname{##1}}%
-    \expandafter\def\csname KV@sphinx@div.important_#1\expandafter\endcsname
-        \the\toks0\expandafter{%
-        \csname spx@opt@heavyimportanttrue\expandafter\expandafter\expandafter\endcsname
-        \csname KV@sphinx@div.important_#1\endcsname{##1}}%
-    \expandafter\def\csname KV@sphinx@div.tip_#1\expandafter\endcsname
-        \the\toks0\expandafter{%
-        \csname spx@opt@heavytiptrue\expandafter\expandafter\expandafter\endcsname
-        \csname KV@sphinx@div.tip_#1\endcsname{##1}}%
-  \fi
-  \spx@tempa
+  div.hint_title-background-TeXcolor=sphinx-success-title-bgcolor,
+  div.hint_title-foreground-TeXcolor=sphinx-success-title-fgcolor,
+  div.tip_title-background-TeXcolor=sphinx-success-title-bgcolor,
+  div.tip_title-foreground-TeXcolor=sphinx-success-title-fgcolor,
+  div.seealso_title-background-TeXcolor=sphinx-success-title-bgcolor,
+  div.seealso_title-foreground-TeXcolor=sphinx-success-title-fgcolor,
+%
+  div.important_title-background-TeXcolor=sphinx-warning-title-bgcolor,
+  div.important_title-foreground-TeXcolor=sphinx-warning-title-fgcolor,
+  div.caution_title-background-TeXcolor=sphinx-warning-title-bgcolor,
+  div.caution_title-foreground-TeXcolor=sphinx-warning-title-fgcolor,
+  div.warning_title-background-TeXcolor=sphinx-warning-title-bgcolor,
+  div.warning_title-foreground-TeXcolor=sphinx-warning-title-fgcolor,
+%
+  div.attention_title-background-TeXcolor=sphinx-error-title-bgcolor,
+  div.attention_title-foreground-TeXcolor=sphinx-error-title-fgcolor,
+  div.danger_title-background-TeXcolor=sphinx-error-title-bgcolor,
+  div.danger_title-foreground-TeXcolor=sphinx-error-title-fgcolor,
+  div.error_title-background-TeXcolor=sphinx-error-title-bgcolor,
+  div.error_title-foreground-TeXcolor=red,
+%
+% TODO: implement todo (sic)
+%
+% Icon defaults.  These names are copied over from sphinx13.css
+% and their actual LaTeX definitions are above and emulate the svg
+% icons, if fontawesome5 was found, else they expand to do nothing.
+  div.note_title-icon=\@nameuse{sphinx-icon-pencil},
+  div.hint_title-icon=\@nameuse{sphinx-icon-question},
+  div.tip_title-icon=\@nameuse{sphinx-icon-info},
+  div.seealso_title-icon=\@nameuse{sphinx-icon-info},
+  div.important_title-icon=\@nameuse{sphinx-icon-flame},
+  div.caution_title-icon=\@nameuse{sphinx-icon-spark},
+  div.warning_title-icon=\@nameuse{sphinx-icon-warning},
+  div.attention_title-icon=\@nameuse{sphinx-icon-warning},
+  div.danger_title-icon=\@nameuse{sphinx-icon-spark},
+  div.error_title-icon=\@nameuse{sphinx-icon-failure},
```

Comment:
(this second comment somehow got lost from my review)
- hint and (perhaps less so) tip could use rather "light bulb"
- seealso should probably use "share" (which is very much like our `more.png`)
- I could not find an icon such as "spark" in fontawesome5 but probably I simply did not check well
- error uses sphinx-icon-failure defined as "skull-crossbones" but it should be "times-circle" to imitate sphinx13.css